sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -114,7 +115,8 @@ static bool PrintPiTrace = false;
 
 static void PiTrace(std::string TraceString) {
   if (PrintPiTrace) {
-    std::cout << TraceString << std::endl;
+    printf("%s\n", TraceString.c_str());
+    fflush(stdout);
```

Comment:
Why do we need to change anything in source files? The PR title says <iostream> is removed from headers files.

NOTE: in addition to source files we can keep plug-in header files untouched, if we update only header files included into `sycl/sycl.hpp`.
The same applies to all headers in `source` directory. They are not included to `sycl/sycl.hpp` as well.

Reply:
I'm sorry the PR title is a little misleading. This does remove \<iostream\> headers form the source files as well. So according to LLVM coding standards \<iostream\> headers are forbidden in 'library files' (https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden). I interpreted 'library files' to include both the headers and source files.